### PR TITLE
Licence and readme

### PR DESCRIPTION
**Pull request by mike29736**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# GOV.UK SLI Collector
+
+Collects data about GOV.UK services and produces SLI metrics for use in SLO dashboards.
+
+## Nomenclature
+
+**SLI**
+
+> An SLI is a service level indicator—a carefully defined quantitative measure of some aspect of the level of service that is provided.[^1]
```
Comment (jkempster34) [inline README.md:9]: hyphen here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+**SL0**
+
+> An SLO is a service level objective: a target value or range of values for a service level that is measured by an SLI.[^1]
```
Comment (jkempster34) [inline README.md:13]: but colon here!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Collects data about GOV.UK services and produces SLI metrics for use in SLO dashboards.
+
+## Nomenclature
+
+**SLI**
+
+> An SLI is a service level indicator—a carefully defined quantitative measure of some aspect of the level of service that is provided.[^1]
+
+**SL0**
```
Comment (jkempster34) [inline README.md:11]: I think this O is the number 0!

File: README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# GOV.UK SLI Collector
```
Comment (jkempster34) [inline README.md:1]: Like the name!

**Review by jkempster34 — APPROVED** [review]
LGTM. Left some comments about grammar

File: README.md (hunk 3 above)
Comment (mike29736) [reply README.md]: You have a keen eye for details

File: README.md (hunk 2 above)
Comment (mike29736) [reply README.md]: Those inconsistencies were straight from the pages of the SRE Book!

I've chopped those first parts off of both definitions and included the full terms with the acronyms now.

**Pull request merged**